src/interface/src/app/map/map-manager.ts

```diff
@@ -23,23 +23,34 @@ export class MapManager {
   maps: Map[];
   popupService: PopupService;
 
+  boundaryGeoJsonCache = new Map<string, GeoJSON.GeoJSON>();
```

Comment:
Should this cache be in the mapService instead?

Reply:
I wanted to avoid exposing services to the helper class as much as possible. Do you think there's any point in doing that? MapManager could do pretty much everything and MapComponent wouldn't need to call MapService at all, but that just felt a bit wrong to me for some reason.